### DISCUSSION TITLE
响应吐字慢

### DISCUSSION THREAD

**Discussion by nikissXI** [linweiyuan/go-chatgpt-api#137] [Q&A]
近期发现如果反代运行久了，吐字会变慢，我重启反代容器吐字就快了。大佬晓得什么原因不

**Comment by linweiyuan** [comment]
讲道理的话，没有关系，程序只是转发，好比是个水龙头开到最大，不同时间水压大小取决于上游

**Reply by nikissXI** [reply]
但是，回答很卡的时候，重启反代api立刻就好了。如果没思路的话，我直接定期重启吧
